tests/spec/bool_series_ops/test_conversion.py

```diff
@@ -0,0 +1,25 @@
+from ..tables import p
+
+
+title = "Convert a float value"
+
+table_data = {
+    p: """
+            | b1
+        --+----
+        1 |  T
+        2 |
+        3 |  F
+        """,
+}
```

Comment:
tiny nit, but can we line this up?

```suggestion
    p: """
          | b1
        --+----
        1 |  T
        2 |
        3 |  F
        """,
}
```